src/ChatWindow.test.js

```diff
@@ -122,10 +162,16 @@ it("handles the user's form submission with request to Alexa and populates the s
     senderName: alexa.name
   });
 
-  testOnUserRequestToAlexaSubmitHandling(expectedAlexaResponse, done);
+  testOnUserRequestToAlexaSubmitHandling(
+    mockAuthenticationInfo,
+    expectedAlexaResponse,
+    done
+  );
 });
 
 it("handles the case when AVS throws an error in response to a user request. We should populate the state with the user request and a canned response.", done => {
+  const mockAuthenticationInfo = { access_token: "mockAccessToken" };
```

Comment:
Naming doesnt sound right. Is it a good practice to override 'mock' to constant variables? Naming the constant as authenticationInfo and the value of the property as "some_access_token" suffices, IMO.

Reply:
Applies to mockuserRequestToAlexa too.

Reply:
Done.